src/google/protobuf/compiler/objectivec/objectivec_helpers.h

```diff
@@ -133,6 +133,22 @@ enum ObjectiveCType {
   OBJECTIVECTYPE_MESSAGE
 };
 
+template<class TDescriptor>
+string GetOptionalDeprecatedAttribute(const TDescriptor* descriptor, bool preSpace = true, bool postSpace = false) {
```

Comment:
Looks like if we change postSpace to postNewline and add a newline, then a Message would generate

```
DEPRECATED_ATTRIBUTE
@intreface MyMessage: ...
```

Which is more line with what Apple does (and hopefully doesn't screw up header docs).